controller/bookmark.js

```diff
@@ -81,6 +82,45 @@ module.exports = {
     next()
   },
 
+  // @route   GET api/bookmarks/tag/// /dates/?startDate=2018.12.01&endDate=2019.05.15,
```

Comment:
this route desc looks a bit confused 😛

Reply:
Thank for that I have fixed the commenting!